commit d9dc468abddbd210779df842127d56dedbdaaf32
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2019-05-07T18:02:54+02:00 (5 months ago)
    
    TRITON-1648 triton-logarchiver shouldn't left workspace "dirty" after a successful build

diff --git a/Makefile b/Makefile
index 39489d6..61afcf2 100644
--- a/Makefile
+++ b/Makefile
@@ -50,6 +50,7 @@ include ./deps/eng/tools/mk/Makefile.node_modules.defs
 ROOT		:= $(shell pwd)
 RELEASE_TARBALL	:= $(NAME)-pkg-$(STAMP).tar.gz
 RELSTAGEDIR	:= /tmp/$(NAME)-$(STAMP)
+DISTCLEAN_FILES += RELEASE_TARBALL
 
 # triton-origin-multiarch-18.1.0
 BASE_IMAGE_UUID = 04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f
